src/index.html

```diff
@@ -5,9 +5,376 @@
+            alt="Logo"
+            class="header__logo"
+          >
+        </a>
+        <nav class="header_mobile">
+          <div class="menuToggle">
+            <input
+              type="checkbox"
+              id="menu"
+              class="menuToggle__box"
+            >
+            <span class="menuToggle__span"></span>
+            <span class="menuToggle__span"></span>
+            <span class="menuToggle__span"></span>
+            <ul class="menuToggle__list">
+              <li class="menuToggle__item">
+                <a href="" class="menuToggle__link">home</a>
+              </li>
+              <li class="menuToggle__item">
+                <a href="#about-us" class="menuToggle__link">about us</a>
+              </li>
+              <li class="menuToggle__item">
+                <a href="#gallery" class="menuToggle__link">gallery</a>
+              </li>
+              <li class="menuToggle__item">
+                <a href="#consultation" class="menuToggle__link">consultation</a>
+              </li>
+              <li class="menuToggle__item">
+                <a href="#testimonials" class="menuToggle__link">testimonials</a>
+              </li>
+              <li class="menuToggle__item">
+                <a href="#contact" class="menuToggle__link">contacts</a>
+              </li>
+              <div class="menuToggle__top">
+                <a href="#" class="menuToggle__phone">+1 234 555-55-55</a>
+                <a href="#" class="menuToggle__call">call to order</a>
+              </div>
+              <a href="#" class="menuToggle__logo">
+                <img
+                  src="images/logo.png"
+                  alt="Logo"
+                  height="48"
+                  width="73"
+                >
+              </a>
+            </ul>
+          </div>
+        </nav>
+        <div class="header-bottom">
+          <h1 class="header__title">Miami<br>
+            Condo Kings
+          </h1>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item nav__item--is-active">
+                <a href="" class="nav__link">home</a>
+              </li>
+              <li class="nav__item">
+                <a href="#about-us" class="nav__link">about us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#gallery" class="nav__link">gallery</a>
+              </li>
+              <li class="nav__item">
+                <a href="#consultation" class="nav__link">consultation</a>
+              </li>
+              <li class="nav__item">
+                <a href="#testimonials" class="nav__link">testimonials</a>
+              </li>
+              <li class="nav__item">
+                <a href="#contact" class="nav__link">contacts</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <main class="content">
+      <section class="aboutUs container" id="about-us">
```

Comment:
do not use camelcase in class names
```suggestion
      <section class="about container" id="about-us">
```